scripts/oncotree_to_oncotree.py

```diff
@@ -221,11 +225,15 @@ def translate_oncotree_codes(source_file_mapped_list, source_oncotree, target_on
 # 3) multiple directly mapped options (w/ or w/o children), False
 # 4) multiple related options (closest parents/children, don't include children), False
 def get_oncotree_code_options(source_oncotree_code, source_oncotree, target_oncotree, is_backwards_mapping):
-    if source_oncotree_code in ["N/A", "", "NA"]:
+    if source_oncotree_code in PASSTHROUGH_ONCOTREE_CODE_LIST:
         return source_oncotree_code, True
     if source_oncotree_code not in source_oncotree:
-        print >> sys.stderr, "ERROR: Oncotree code (%s) can not be found in source oncotree. Please verify source version." % (source_oncotree_code)
-        sys.exit(1)
+        GLOBAL_LOG_MAP[source_oncotree_code][CHOICES_FIELD].append("???")
```

Comment:
should put a conditional here of whether the code is already logged -- otherwise you could end up with something like 

{"FAKECODE": {"CHOICES":["???", "???", "???", "???"}}

and I think your list comprehension when printing out the logs files has a conditional which only prints codes where the CHOICES list is length 1

Reply:
good catch .. adding check

Reply:
actually I accomplished this with:
GLOBAL_LOG_MAP[source_oncotree_code][CHOICES_FIELD] = ["???"]

Reply:
not sure if github is glitching but I see 'append' and not '='

Reply:
not sure if github is glitching but I see 'append' and not '='